python/lsst/validate/drp/calcsrd/__init__.py

```diff
@@ -8,3 +8,4 @@
 from .amx import AMxMeasurement  # NOQA
 from .afx import AFxMeasurement  # NOQA
 from .adx import ADxMeasurement  # NOQA
```

Comment:
#NOQA for `from .tex` is in keeping with previous practice for these SRD metrics.  Happy to consider overall #NOQA use in future ticket.